src/modules/api.js

```diff
@@ -1,24 +1,34 @@
 export default class Api {
-    #fetchResults= async () => {
-      const { results } = await (await fetch('https://pokeapi.co/api/v2/pokemon?limit=6&offset=10')).json();
-      return results;
-    }
+  POKEMON_URL = 'https://pokeapi.co/api/v2/pokemon';
 
-    #fetchPockemon=async (url) => {
-      const result = {};
-      const data = await (await fetch(url)).json();
-      result.url = data.sprites.other.dream_world.front_default;
-      result.id = data.id;
-      return result;
-    }
+  #fetchResults = async () => {
+    const { results } = await (await fetch(`${this.POKEMON_URL}?limit=6&offset=10`)).json();
+    return results;
+  };
 
-    getDisplayItems = async () => {
-      const promises = [];
-      const results = await this.#fetchResults();
-      results.forEach((result) => {
-        promises.push(this.#fetchPockemon(result.url));
-      });
-      const items = await Promise.all(promises);
-      return items;
-    }
-}
\ No newline at end of file
+  #fetchPockemon = async (url) => {
+    const result = {};
+    const data = await (await fetch(url)).json();
+    result.url = data.sprites.other.dream_world.front_default;
+    result.id = data.id;
+    result.name = data.name;
+    result.height = data.height;
+    result.weight = data.weight;
+    return result;
+  };
+
+  getDisplayItems = async () => {
+    const promises = [];
+    const results = await this.#fetchResults();
+    results.forEach((result) => {
+      promises.push(this.#fetchPockemon(result.url));
+    });
+    const items = await Promise.all(promises);
+    return items;
+  };
+
+  getPokemonFromId = async (idPokemon) => {
+    const data = await this.#fetchPockemon(`${this.POKEMON_URL}/${idPokemon}`);
```

Comment:
Please insert the last back-slash in order to fix strict CORS erros
```suggestion
    const data = await this.#fetchPockemon(`${this.POKEMON_URL}/${idPokemon}/`);
```

Reply:
Ok, thank you! I didn't know that before. I'll correct it